src/ceph-osd-prestart.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
can we instead remove the bashism in this script? as `ceph-osd` does not depend on "bash".